src/Tizen.NUI/src/devel/Lite/UIColorCosntants.cs

```diff
@@ -22,59 +22,757 @@ namespace Tizen.NUI
 {
```

Comment:
There's typo in filename.
`UIColorCosntants.cs` -> `UIColorConstants.cs`

Reply:
Oh..! very critical issue! Fix done.